SmellDetector/Controllers/DetectionService.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using SmellDetector.DTO;
+
+public class DetectionService
```

Comment:
For now we can leave the DetectionService in the Controllers folder, but note that this is not a good location for this service. I imagine you looked to the RepositoryCompiler for inspiration, where the service is misplaced in the Controllers folder as well.

Just make sure you don't follow this poor practice in your PSW project :)

Reply:
Yes, i looked at RepositoryCompiler and i thought we are putting Services in Controllers folder for some reason 😄 

Ofc i will not use this practice in our PSW project 😄